legend-engine-xt-relationalStore-grammar/src/main/java/org/finos/legend/engine/language/pure/compiler/toPureGraph/HelperRelationalBuilder.java

```diff
@@ -306,12 +306,9 @@ public static Relation getRelation(Database db, final String _schema, final Stri
         return getRelation(db, _schema, _table, SourceInformation.getUnknownSourceInformation());
     }
 
-    private static Relation getRelation(Database db, final String _schema, final String _table, SourceInformation sourceInformation)
+    public static Relation getRelation(Database db, final String _schema, final String _table, SourceInformation sourceInformation)
```

Comment:
Expose method that receives source information to allow better error messages on extensions.